[ Docs] Update bare metal setup instructions (#2281)
paperless-ngx/paperless-ngx: docs/setup.md

        extension](https://code.djangoproject.com/wiki/JSON1Extension) is
        enabled. This is usually the case, but not always.

4.  Create a system user with a new home folder under which you wish
    to run paperless.

    ```shell-session
    adduser paperless --system --home /opt/paperless --group
    ```

5.  Get the release archive from
    <https://github.com/paperless-ngx/paperless-ngx/releases> for example with 
    ```shell-session
    curl -O -L https://github.com/paperless-ngx/paperless-ngx/releases/download/v1.10.2/paperless-ngx-v1.10.2.tar.xz
    ```
    Extract the archive with 
    ```shell-session
    tar -xf paperless-ngx-v1.10.2.tar.xz
    ```

>     and copy the content to the 

Reviewer: ```suggestion
    and copy the contents to the 
```

    home folder of the user you created before (`/opt/paperless`).
    

>     Optional: If you clone the git repo as it is, you also have to

Reviewer: ```suggestion
    Optional: If you cloned the git repo, you will have to
```

    sudo -Hu paperless python3 manage.py runserver
    ```

>     and pointing your browser to <http://localhost:8000/> if 
>     using the browser from the same devices as paperless is installed.

Reviewer: ```suggestion
    and pointing your browser to http://localhost:8000 if 
    accessing from the same devices on which paperless is installed.
```

        For instructions on how to use nginx for that,
        [see the instructions below](/setup#nginx).

    !!! warning

        If celery won't start (check with 
        `sudo systemctl status paperless-task-queue.service` for 

>         paperless-task-queue.service und paperless-scheduler.service

Reviewer: ```suggestion
        paperless-task-queue.service and paperless-scheduler.service
```
